src/main/java/com/revature/models/Curriculum.java

```diff
@@ -33,40 +38,43 @@
 	
 	@NotNull
```

Comment:
not sure about the spacing here but be sure to have a maximum of only one line of space between the fields or methods of the pojo